dev/tasks/tasks.yml

```diff
@@ -498,7 +499,9 @@ tasks:
       macos_deployment_target: "11.0"
     artifacts:
       - pyarrow-{no_rc_version}-{{ python_tag }}-{{ python_tag }}-macosx_11_0_arm64.whl
+{% endfor %}
 
+{% for python_version, python_tag in [("3.9", "cp39"), ("3.10", "cp310")] %}
```

Comment:
I think we could also stop providing universal wheels altogether for new pyarrow versions? (and not just stop providing them for the python 3.11 wheels?)

Reply:
Done